### ISSUE TITLE
Top Bar: Hiver Extensions with Simplify

### ISSUE THREAD

**Issue by manuel-singh** [leggett/simplify#764]
https://www.loom.com/share/a7a12dbd4b024416acc8380503e620b5?sid=72f7e451-1ff4-4893-bb86-13746afc67fc

**Comment by leggett** [comment]
Hiver made some changes and this is indeed broken on Simplify Gmail v2. 

If you switch to [Simplify Gmail v3](https://beta.simpl.fyi), I have fixed it there. I'm working to get those fixes to the v2 version (or better, launch v3 to all).

**Comment by Fammy** [comment]
> or better, launch v3 to all

Yes please, says this Firefox user! The hardest part of switching from Chrome to Firefox was losing bundles. (Sorry to hijack this issue)

**Comment by leggett** [comment]
Definitely working towards that. And now that Firefox is FINALLY adding support for :has() in v120, I won't have any more excuses. :)
